python/core/auto_generated/symbology/qgsmarkersymbollayer.sip.in

```diff
@@ -923,14 +924,14 @@ Sets the font ``family`` for the font which will be used to render the point.
 .. seealso:: :py:func:`fontFamily`
 %End
 
-    QChar character() const;
+    QString character() const;
 %Docstring
 Returns the character used when rendering points.
 
 .. seealso:: :py:func:`setCharacter`
 %End
 
-    void setCharacter( QChar ch );
+    void setCharacter( QString chr );
```

Comment:
chr -> characters

Reply:
I think I can't change that, it'd break the python API (i.e. `setCharacter( chr='d' )` )